### PR TITLE
User api call backend base

### PR DESCRIPTION
**Pull request by pruthvichivukula**

### CODE REVIEW

File: src/main/java/SpringJPA/Model/User.java [hunk 1]
```diff
@@ -29,7 +33,18 @@ public User(String user, String pass, String role, String mail) {
...
+    public User(String user, String pass, String role, long apiCallCount, long apiCallLimit) {
+        this.username = user;
+        this.password = pass;
+        this.role = role;
+        this.apiCallCount = apiCallCount;
+        this.apiCallLimit = apiCallLimit;
+    }
+
+    public void incrementApICallCount(){
+        this.apiCallCount++;
```
Comment (namansethi) [inline src/main/java/SpringJPA/Model/User.java]: Should there be a maximum call count for APIs?

Comment (pruthvichivukula) [reply src/main/java/SpringJPA/Model/User.java]: yes, you are correct, I will implement that change now

Comment (pruthvichivukula) [reply src/main/java/SpringJPA/Model/User.java]: I have made the change, please review

**Review by namansethi — APPROVED** [review]
All issues dealt with, good to go.

**Pull request merged**